Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/User/UserGroup/ByUserGroup.tsx

```diff
@@ -85,8 +87,8 @@ const ByUser: Application.Types.iByComponent = (props) => {
     return (
         <div style={{ width: '100%', height: '100%' }}>
             <LoadingScreen Show={pageStatus === 'loading'} />
-            <SearchBar<ISecurityGroup> CollumnList={defaultSearchcols} SetFilter={(flds) => dispatch(SecurityGroupSlice.DBSearch({ filter: flds }))}
-                Direction={'left'} defaultCollumn={{ label: 'Name', key: 'DisplayName', type: 'string', isPivotField: false }} Width={'50%'} Label={'Search'}
+            <SearchBar<ISecurityGroup> CollumnList={filterableList} SetFilter={(flds) => dispatch(SecurityGroupSlice.DBSearch({ sortField, ascending, filter: flds }))}
+                Direction={'left'} defaultCollumn={{ label: 'Name', key: 'Name', type: 'string', isPivotField: false }} Width={'50%'} Label={'Search'}
```

Comment:
This may be a question fro @gcsantos-gpa but do we have a typing issue? If we use `Searchbar<T>` shouldn't it type `defaultCollum` to  `Search.IField<T>` and use `key: keyof T`? That should have caught this issue?